evo/tools/file_interface.py

```diff
@@ -199,7 +199,9 @@ def read_bag_trajectory(bag_handle, topic):
         raise FileInterfaceException(
             "unsupported message type: {}".format(msg_type))
     stamps, xyz, quat = [], [], []
-    for topic, msg, t in bag_handle.read_messages(topic):
+    for topic, msg, _ in bag_handle.read_messages(topic):
```

Comment:
I made it completely hidden because it's not (and shouldn't) be used.

Reply:
Great! I had left it because I wasn't sure if there was a use case for it and if we should be adding a flag to the method to allow the user to choose. I couldn't think of one though, so I agree with hiding it.